checkstyle-tester/diff.groovy

```diff
@@ -861,3 +861,7 @@ class CheckstyleReportInfo {
         this.commitTime = commitTime
     }
 }
+
+class Constants {
+    static final TWO = 2
```

Comment:
Do we really need to make our code terrible?
I would rather keep suppression or disable rule